changelog/@unreleased/pr-4483.v2.yml

```diff
@@ -0,0 +1,8 @@
+type: improvement
+improvement:
+  description: "Add ability to make persister reusable and not to create a new
+    instance on each hydration. The default behaviour is to create
+    a new instance of persister on each hydration to avoid issues with persisters
+    which are stateful."
```

Comment:
Might be worth mentioning that this is done through specifying the `@Reusable` annotation, and that if users want to enable this they will need to regenerate their schemas.

Also we should say that the persisters are only reusable if they are stateless and thread-safe.